patches/chrome-android-BUILD.gn.patch

```diff
@@ -11,3 +11,21 @@ index 02f17d2b4e93d27964fe6663c4feddccc9634619..bd0459db766702d7bc0127914e687421
  test_manifest_package = "org.chromium.chrome.tests"
  
  chrome_public_jinja_variables = default_chrome_public_jinja_variables +
+@@ -1061,7 +1061,8 @@ jinja_template_resources("chrome_public_apk_template_resources") {
+     "java/res_template/xml/file_paths.xml",
+     "java/res_template/xml/launchershortcuts.xml",
+     "java/res_template/xml/searchable.xml",
+-    "java/res_template/xml/syncadapter.xml",
```

Comment:
I'm thinking what we really want to do here is to override GoogleServicesManager to do nothing because that will disable the SigninController and the SyncController. wdyt?

Reply:
actually these get called from a lot of places and https://github.com/brave/brave-core/pull/2852/files#diff-bd3dc461dbf2e7aa0c0bc537b1f90530R9 already overrides the SigninManager so I think it makes more sense to override `ChromeSigninController` and `SyncController` in a similar way

Reply:
I don't think we'll want to disable the SyncController because we're actually using the sync service now with @darkdh's changes so I think we'll want our own BraveSyncController in android